app.py

```diff
@@ -105,6 +104,22 @@ def education():
 
         return jsonify({'id': new_id}), 201
     
+    if request.method == 'DELETE':
+        index = request.args.get("index")
+        # check index is valid number
+        if index is not None:
+            if not index.isnumeric():
+                return jsonify({"error": "Index must be a number"}), 400
+            
+            if 0 < int(index) <= len(data["education"]):
```

Comment:
it needs to be <= because the ids are 1 indexed when the lists are 0 indexed, which means that when a new education is created, it would have the id equal to the length of the list

Reply:
Oh, makes sense!